app/view_objects/articles/social_image.rb

```diff
@@ -32,7 +32,7 @@ def url
     attr_reader :article
 
     def legacy_article_social_image
-      cache_key = "article-social-img-#{article}-#{article.updated_at}-#{article.comments_count}"
+      cache_key = "article-social-img-#{article}-#{article.updated_at&.rfc3339}-#{article.comments_count}"
```

Comment:
I don't think we need the `&` for `article.updated_at`

Reply:
Resolved